internal/credential/vault/repository_credential_store.go

```diff
@@ -116,6 +117,11 @@ func (r *Repository) CreateCredentialStore(ctx context.Context, cs *CredentialSt
 		return nil, err
 	}
 
+	runJobsInterval := r.scheduler.GetRunJobsInterval()
+	if token.expiration <= runJobsInterval {
+		return nil, errors.Wrap(ctx, fmt.Errorf("scheduler interval must be greater than token ttl. scheduler jobs interval value: %s seconds", strconv.FormatFloat(runJobsInterval.Seconds(), 'f', -1, 64)), op)
```

Comment:
Do we have other examples in Boundary where we print a duration? Getting it in seconds is probably fine, but we could just print out using golang's default duration print.

Reply:
I found one example where we print the time.Duration variable using the String() func, which I just tested out and it does look nicer.